src/java.security.jgss/share/classes/sun/security/jgss/krb5/Krb5AcceptCredential.java

```diff
@@ -65,16 +63,15 @@ static Krb5AcceptCredential getInstance(final GSSCaller caller, Krb5NameElement
 
         final String serverPrinc = (name == null? null:
             name.getKrb5PrincipalName().getName());
-        final AccessControlContext acc = AccessController.getContext();
 
         ServiceCreds creds = null;
         try {
-            creds = AccessController.doPrivileged(
+            creds = AccessController.doPrivilegedWithCombiner(
```

Comment:
Suggest adding a comment here and other places briefly explaining why doPrivWithCombiner is now needed to preserve the SubjectDomainCombiner.